### ISSUE TITLE
About convergence

### ISSUE THREAD

**Issue by gitDaxian** [OreoChocolate/MUREN#6]
Exciting work！Could you tell how many epochs were trained? I would like to know the convergence.Thinks!

**Comment by OreoChocolate** [comment]
Thank you for your interest in our research. 
We train our model for 100 epochs, and convergence occurs after the learning rate decay.
